### ISSUE TITLE
Deploy Chaincode is not working.

### ISSUE THREAD

**Issue by girishchawla** [olegabu/insurance-fronting#3]
curl -XPOST -d '{"jsonrpc": "2.0", "method": "deploy", "params": {"type": 1,"chaincodeID": {"path": "github.com/olegabu/insurance-fronting/chaincode","language": "GOLANG"}, "ctorMsg": { "args": ["aW5pdA=="] },"secureContext": "citi", "attributes": ["role", "company"]},"id": 0}' http://vp0:7050/chaincode
curl: (6)

 Couldn't resolve host 'vp0'

**Comment by girishchawla** [comment]
Is there some other commands to deploy the chain missing ? When I run deploy.sh, I get peer command not found

**Comment by olegabu** [comment]
Have you set up the blockchain network? Are peers up and running?

**Comment by girishchawla** [comment]
No. Actually I am new to this. Any pointers in terms of URLs or instructions will be much appreciated. Should I deploy the hyperledger fabric on my laptop ?

**Comment by olegabu** [comment]
Yes. Please refer to the Fabric 0.6 docs, not the latest.

**Comment by girishchawla** [comment]
Finally I was able to build the fabric 0.6 and have the vagrant up and running. 

User Name@HP MINGW64 /c/Go/src/github.com/hyperledger/fabric/devenv
$ ls
compile_protos.sh*   images/      setupRHELonZ.sh*
failure-motd.in      limits.conf  setupUbuntuOnPPC64le.sh*
golang_buildcmd.sh*  README.md    Vagrantfile
golang_buildpkg.sh*  setup.sh*

User Name@HP MINGW64 /c/Go/src/github.com/hyperledger/fabric/devenv
$ vagrant port
The forwarded ports for the machine are listed below. Please note that
these values may differ from values configured in the Vagrantfile if the
provider supports automatic port collision detection and resolution.

  7050 (guest) => 7050 (host)
  7051 (guest) => 7051 (host)
  7053 (guest) => 7053 (host)
  7054 (guest) => 7054 (host)
    22 (guest) => 2200 (host)

User Name@HP MINGW64 /c/Go/src/github.com/hyperledger/fabric/devenv
$ vagrant status
Current machine states:

default                   running (virtualbox)

The VM is running. To stop this VM, you can run `vagrant halt` to
shut it down forcefully, or you can run `vagrant suspend` to simply
suspend the virtual machine. In either case, to restart it again,
simply run `vagrant up`.


I still get the "Peer command not found" when I run the deploy.sh  in the deploy directory. Also in the web application if I click on New Policy nothing seems to happen. I also tried the other project you have i.e catbond. It gives the same kind of error when I try to run start.sh or deploy_chaincode.sh
